frontend/src/queries/vault.ts

```diff
@@ -14,6 +14,9 @@ export const GET_USER_VAULTS = gql`
         }
         locks(first: 1000) {
           id
+          geyser {
+            id
+          }
```

Comment:
this will break the crucible integration

Reply:
yea just realized after someone complained on telegram